meinberlin/apps/moderationtasks/templates/meinberlin_moderationtasks/moderation_tasks_form.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n %}
+
+<p class="form-hint">{% blocktrans %}Moderation tasks appear in the detailed view
```

Comment:
detail view, not detailed view. Even though, it's probably like this in the wording file.

Reply:
Right, and `blocktranslate` when we add new templates. Should probably change that in all templates, otherwise I also keep forgetting that.